Author: Erik B. Andersen <erik.b.andersen@gmail.com>
Description: Change lxc-clone to use 'rsync -aH' instead of just 'rsync -a' for cloning to fix Launchpad Bug #1441307.

diff -Naurp lxc.orig/src/lxc/bdev.c lxc/src/lxc/bdev.c
--- lxc.orig/src/lxc/bdev.c
+++ lxc/src/lxc/bdev.c
@@ -93,7 +93,7 @@ static int do_rsync(const char *src, con
 	s[l-2] = '/';
 	s[l-1] = '\0';
 
-	execlp("rsync", "rsync", "-a", s, dest, (char *)NULL);
+	execlp("rsync", "rsync", "-aH", s, dest, (char *)NULL);
 	exit(1);
 }
 
